lib/args.js

```diff
@@ -39,33 +45,29 @@ function buildYargs(args = null) {
 const PR_RE = new RegExp(
   '^https://github.com/(\\w+)/([a-zA-Z.-]+)/pull/' +
   '([0-9]+)(?:/(?:files)?)?$');
+
 function checkAndParseArgs(args) {
-  if (typeof args.r === 'undefined' && typeof args.o !== 'undefined') {
-    args.r = args.o || args.owner;
-    args.o = 'nodejs';
-  }
-  // Fast path: numeric string
-  if (!isNaN(args.identifier)) {
-    return {
-      owner: args.o || args.owner || 'nodejs',
-      repo: args.r || args.repo || 'node',
-      id: +args.identifier
-    };
-  }
-  const match = args.identifier.match(PR_RE);
-  if (match !== null) {
-    if (typeof args.r !== 'undefined' || typeof args.o !== 'undefined') {
-      throw new Error(
-        'Cannot pass second or third argument ' +
-        'when url given as first argument.');
+  const { owner, repo, identifier, file } = args;
```

Comment:
Can we specify the defaults here in destructuring (including an empty string for identifier) instead and then the line below would be `const result = { owner, repo, file };`?